core/schemas/geometry-object.json

```diff
@@ -1,8 +1,59 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://beta.schemas.opengis.net/json-fg/geometry-objects.json",
+  "$id": "https://beta.schemas.opengis.net/json-fg/geometry-object.json",
   "title": "the geometry objects",
   "description": "This JSON Schema is part of JSON-FG version 0.3.0",
+  "type": "object",
+  "oneOf": [
+    {
+      "$ref": "#/$defs/Point"
+    },
+    {
+      "$ref": "#/$defs/MultiPoint"
+    },
+    {
+      "$ref": "#/$defs/LineString"
+    },
+    {
+      "$ref": "#/$defs/MultiLineString"
+    },
+    {
+      "$ref": "#/$defs/Polygon"
+    },
+    {
+      "$ref": "#/$defs/MultiPolygon"
+    },
+    {
+      "$ref": "#/$defs/Polyhedron"
+    },
+    {
+      "$ref": "#/$defs/MultiPolyhedron"
+    },
+    {
+      "$ref": "#/$defs/Prism"
+    },
+    {
+      "$ref": "#/$defs/MultiPrism"
+    },
+    {
+      "$ref": "#/$defs/CircluarString"
```

Comment:
CircularString

Reply:
You are reviewing an outdated status. This has already been fixed.